org.musicbrainz.Picard.json

```diff
@@ -24,7 +22,7 @@
             "PYTHON": "/usr/bin/python3",
             "PIP": "/usr/bin/pip3"
         }
-    },	
+    },
```

Comment:
The pedantic in me would prefer you made this change in its own commit. :stuck_out_tongue:

Reply:
Too late I'm afraid. It's no big deal anyway imho.

Reply:
It's never too late with git. :stuck_out_tongue: